index.d.ts

```diff
@@ -0,0 +1,2 @@
+declare const norwayPostalCodes: Record<string, string>;
```

Comment:
Needs a doc comment

Reply:
Added a basic doc comment. Not sure if there is any particular way to make docstrings for objects.